tools/ocamltex.ml

```diff
@@ -765,7 +765,7 @@ let process_file file =
          spanned the interval %d-%d,@ \
          intersecting with another \"%a\" transform @ \
          on the %d-%d interval.@ \
-         Hind: did you try to elide a code fragment which raised a warning?"
+         @{<hint>Hint@}: did you try to elide a code fragment which raised a warning?"
```

Comment:
Well done for spotting the spelling mistake too!

Reply:
Thank you.